spec/gar_spec.lua

```diff
@@ -2940,6 +2940,11 @@ describe("gar", function()
 			}
 			targetGateway.totalDelegatedStake = qty
 
+			sourceGateway.delegates = nil
+			sourceGateway.vaults = nil
+			targetGateway.delegates = nil
+			targetGateway.vaults = nil
+
```

Comment:
nit: we're losing coverage here. I believe we can instead keep the expectations but use _G